src/landscape/components/LandscapeDefaultHomeCard.js

```diff
@@ -18,52 +18,31 @@ import React from 'react';
 
 import { useTranslation } from 'react-i18next';
 
-import { Alert, Box, Divider, Typography } from '@mui/material';
+import { Link, Typography } from '@mui/material';
 
-import CardActionRouterLink from 'common/components/CardActionRouterLink';
-
-import HomeCard from 'home/components/HomeCard';
+import DefaultHomeCard from 'home/components/DefaultHomeCard';
 
 const LandscapeDefaultHomeCard = () => {
   const { t } = useTranslation();
-
+  const landscapeImage = require(`assets/${t(`landscape.img.src`)}`);
   return (
-    <HomeCard
-      aria-labelledby="landscapes-default-title"
-      sx={{ flexDirection: 'column' }}
+    <DefaultHomeCard
+      item="landscape"
```

Comment:
Suggestion: I think this could be considered an `id` instead of `item`, so `id` here makes more sense to me